src/LieTheory/WeylGroup.jl

```diff
@@ -514,11 +559,96 @@ end
 Return `x` as a list of indices of simple reflections, in reduced form.
 
 This function is right inverse to calling `(W::WeylGroup)(word::Vector{<:Integer})`.
+
+# Examples
+
+```jldoctest
+julia> W = weyl_group(:A, 2)
+Weyl group
+  of root system of rank 2
+    of type A2
+
+julia> x = longest_element(W)
+s1 * s2 * s1
+
+julia> wd = word(x)
+3-element Vector{UInt8}:
+ 0x01
+ 0x02
+ 0x01
+
+julia> W(wd) == x
+true
+```
 """
 function word(x::WeylGroupElem)
   return x.word
 end
 
+@doc raw"""
+    letters(x::WeylGroupElem) -> Vector{UInt8}
+
+Return `x` as a list of indices of simple reflections, in reduced form.
+
+This function is right inverse to calling `(W::WeylGroup)(word::Vector{<:Integer})`.
+
+# Examples
+
+```jldoctest
+julia> W = weyl_group(:A, 2)
+Weyl group
+  of root system of rank 2
+    of type A2
+
+julia> x = longest_element(W)
+s1 * s2 * s1
+
+julia> l = letters(x)
+3-element Vector{UInt8}:
+ 0x01
+ 0x02
+ 0x01
+
+julia> W(l) == x
+true
+```
+"""
+function letters(x::WeylGroupElem)
+  return x.word
```

Comment:
```suggestion
  return Int.(x.word)
```
all other `letters` methods return a `Vector{Int}` and this should be consistent with it.